src/core/ext/filters/http/client/http_client_filter.cc

```diff
@@ -99,7 +99,7 @@ struct channel_data {
 };
 }  // namespace
 
-static grpc_error* client_filter_incoming_metadata(grpc_call_element* elem,
+static grpc_error* client_filter_incoming_metadata(grpc_call_element* /*elem*/,
```

Comment:
I think we can just remove this parameter.

Reply:
Thanks for the detailed reviews! I will get to these comments.